src/recorder/jamcontroller.h

```diff
@@ -73,4 +75,9 @@ class CJamController : public QObject
 
 }
 
+// This must be included AFTER the above definition of class CJamController,
+// because server.h defines CServer, which contains an instance of CJamController,
+// and therefore needs to know the its size.
+#include "../server.h"
```

Comment:
Oh.  Are you sure it can't just forward-declare the class?
```
...
class CJamController;
...
class CSomethingElse
{
  ...
  CJamController JamController;
  ...
}
```

Reply:
AFAIK, that could only work if `JamController` was a pointer rather than a full object.

Reply:
Ahh, right.  All the code used to be part of CServer.  Cut'n'paste job to create CJamController (with some replace-all help) -- but I didn't try to clean up to that extent when Volker ask it to leave :).  Perhaps it should have been a pointer.  Not for now.